src/libraries/Native/Unix/Common/pal_error_common.h

```diff
@@ -12,7 +12,7 @@
 #include <errno.h>
 #include <netdb.h>
 
-// ENODATA is not defined in FreeBSD 10.3 but is defined in 11.0
+// ENODATA is not defined in FreeBSD
 #if defined(__FreeBSD__) & !defined(ENODATA)
```

Comment:
Since we're no longer targetting these old verisons, do we even need the this guard anymore?

Reply:
So should I just remove `& !defined(ENODATA)` with the expectation it can never be present? I wonder if the original contributor had ENODATA defined by a package they had installed?

Reply:
I suspect that the original contributor was me, but I honestly don't remember why I did it this way. We can remove it if it's no longer useful (or if it never was).